dame_flame/post_matching.py

```diff
@@ -0,0 +1,167 @@
+#%% Imports
+import pandas as pd
+import numpy as np
+import dame_flame
+import time
+from . import DAME_FLAME
+
+#%% Original algorithm speed test
+df = pd.read_csv("data/data.csv")
+
+start_original = time.time()
+result = dame_flame.DAME_FLAME.FLAME(input_data=df, holdout_data = df,verbose=0,treatment_column_name = 'treated', outcome_column_name = 'outcome',repeats = True)
+end_original = time.time()
+print(end_original - start_original)
+
+#%% Modified algorithm speed test
+df = pd.read_csv("data/data.csv")
+
+start_new = time.time()
+result_new = DAME_FLAME.FLAME(input_data=df, holdout_data = df,verbose=0,treatment_column_name = 'treated', outcome_column_name = 'outcome',repeats = True)
+end_new = time.time()
+print(end_new - start_new)
+
+#%% MG function
+def MG(return_df, unit_ids, input_data):
+    '''
+    This function returns the main matched groups for all specified unit
+    indices
+
+    Args:
+        return_df : output of DAME or FLAME
+        unit_id : units for which CATE will be computed
+        input_data : matching data
+        treatment_column_name : name of column containing treatment information
+        outcome_column_name : name of column containing outcome information
+    '''
+    #accept int or list
+    if type(unit_ids) is int:
+        unit_ids = [unit_ids]
+    #define relevant output variables
```

Comment:
nitpicky thing but also for consistent formatting, can we stick with  comments being hash + space + start of comment? I tried (and sometimes was also lazy) to stick with Google's Python Style Guide, so that's where stylistic conventions I'm recommending you change come from:

https://google.github.io/styleguide/pyguide.html

But these are great comments content wise so if you did a similar thing in the FLAME algorithm that would be great!